src/OperationalInsights/OperationalInsights/Tables/CreateAzureOperationalInsightsRestoreTableCommand.cs

```diff
@@ -0,0 +1,70 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.OperationalInsights.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.OperationalInsights.Tables
+{
+    [Cmdlet("Create", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "OperationalInsightsRestoreTable", SupportsShouldProcess = true), OutputType(typeof(PSTable))]
+    public class CreateAzureOperationalInsightsRestoreTableCommand : OperationalInsightsBaseCmdlet
+    {
+        [Parameter(Position = 0, ParameterSetName = ByWorkspaceName, Mandatory = true, ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The resource group name.")]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(Position = 1, ParameterSetName = ByWorkspaceName, Mandatory = true, ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The name of the workspace that will contain the storage insight.")]
+        [ValidateNotNullOrEmpty]
+        public string WorkspaceName { get; set; }
+
+        [Parameter(Position = 2, Mandatory = true, ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The table name.")]
+        [ValidateNotNullOrEmpty]
+        public string TableName { get; set; }
+
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The timestamp to start the restore from (UTC).")]
+        [ValidateNotNullOrEmpty]
+        public string StartRestoreTime { get; set; }
+
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The timestamp to end the restore by (UTC).")]
+        [ValidateNotNullOrEmpty]
+        public string EndRestoreTime { get; set; }
+
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The table to restore data from.")]
+        [ValidateNotNullOrEmpty]
+        public string RestoreSourceTable { get; set; }
```

Comment:
is this a name of the source name or resource id? please make it clear in the parameter name

Reply:
the help message is the same as in the swagger: https://review.docs.microsoft.com/en-us/rest/api/loganalytics/tables/create-or-update?branch=main#restoredlogs
renamed the parameter from "RestoreSourceTable"  to "SourceTable" so it will be the same as in the swagger